libr/core/project.c

```diff
@@ -707,13 +703,14 @@ R_API bool r_core_project_save(RCore *core, const char *prj_name) {
 		RList *paths = r_list_new ();
 		if (paths) {
 			if (r_list_append (paths, prj_dir)) {
-				if (!rvc_git_commit (core, core->prj->rvc,
+				if (!core->prj->rvc->commit (core->prj->rvc,
```

Comment:
function pointers should be opaque to the users. otherwise theres no sense in providing public apis. just keep this call as it was before, and pass core->prj->rvc as argument